src/addons/dragAndDrop/withDragAndDrop.js

```diff
@@ -1,16 +1,18 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 import { DragDropContext } from 'react-dnd'
+import { HTML5Backend } from "react-dnd-html5-backend-14";
+import { DndProvider, useDragDropManager } from "react-dnd-14";
 import cn from 'classnames'
-
 import { accessor } from '../../utils/propTypes'
 import DraggableEventWrapper from './DraggableEventWrapper'
 import { DroppableDayWrapper, DroppableDateCellWrapper } from './DropWrappers'
 
+const shouldUseDragDropContext14 = window.Pulse.features.calendar_dnd_14;
```

Comment:
```suggestion
const shouldUseDragDropContext14 = window?.Pulse.features.calendar_dnd_14;
```

Reply:
I tried, but it doesnt work with ? in this package
I thought its not a big deal since it shouldnt really even happen no ?